taxonomy/forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+Taxonomy forms
+"""
+
+from django.forms import (
+    ModelForm,
+    TextInput,
+    CheckboxSelectMultiple,
+    RadioSelect,
+)
+
+from .models import Vocabulary
+
+
+class VocabularyForm(ModelForm):
```

Comment:
There are descriptions in the screenshot for each field which aren't included in this form but maybe that can be done via CSS? Not sure